astroquery/exoplanets/exoplanet_archive.py

```diff
@@ -0,0 +1,82 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import json
+import os
+from astropy.utils.data import download_file
+from astropy.io import ascii
+import astropy.units as u
+
+__all__ = ['query_exoplanet_archive_catalog']
+
+EXOPLANETS_CSV_URL = ('http://exoplanetarchive.ipac.caltech.edu/cgi-bin/'
+                      'nstedAPI/nph-nstedAPI?table=exoplanets')
+EXOPLANETS_TABLE = None
+PARAM_UNITS = None
+TIME_ATTRS = {'rowupdate': 'iso'}
+BOOL_ATTRS = ('pl_kepflag', 'pl_ttvflag', 'pl_k2flag', 'st_massblend',
+              'st_optmagblend', 'st_radblend', 'st_teffblend')
+
+
+def exoplanet_archive_units():
+    """
+    Dictionary of exoplanet parameters and their associated units.
+    """
+    global PARAM_UNITS
+
+    if PARAM_UNITS is None:
+        module_dir = os.path.dirname(os.path.abspath(__file__))
+        units_file = open(os.path.join(module_dir, 'data',
+                                       'exoplanet_nexsci_units.json'))
+        PARAM_UNITS = json.load(units_file)
+
+    return PARAM_UNITS
+
+
+def query_exoplanet_archive_catalog(cache=True, show_progress=True):
+    """
+    Download (and optionally cache) the NExScI Exoplanet Archive Confirmed
+    Planets table [1]_.
+
+    Parameters
+    ----------
+    cache : bool (optional)
+        Cache exoplanet table to local astropy cache? Default is `True`.
+    show_progress : bool (optional)
+        Show progress of exoplanet table download (if no cached copy is
+        available). Default is `True`.
+
+    Returns
+    -------
+    table : `~astropy.table.Table`
+        Table of exoplanet properties.
+
+    References
+    ----------
+    .. [1] http://exoplanetarchive.ipac.caltech.edu/index.html
```

Comment:
This looks like it formats right when I build locally, so I think it's fine?

![image](https://cloud.githubusercontent.com/assets/143715/19621259/3aa05040-984b-11e6-8f39-b5d4ec32fb13.png)